### PR TITLE
Add Heroku support (#213)

### PR DESCRIPTION
**Pull request by vikmeup**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "addons": [
+    "heroku-postgresql:hobby-dev",
+    "heroku-redis:hobby-dev"
+  ],
+  "environments": {
+    "review": {
+      "addons": [
+        {
+          "plan":"heroku-postgresql:hobby-dev",
+          "as":"STORAGE_POSTGRES"
```
Comment (maximilian-spiridonov) [inline app.json:17]: ~~STORAGE_POSTGRES~~ => STORAGE_POSTGRES_URL

File: app.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        },
+        {
+          "plan":"heroku-redis:hobby-dev",
+          "as":"STORAGE_REDIS"
```
Comment (maximilian-spiridonov) [inline app.json:21]: ~~STORAGE_REDIS~~ =>STORAGE_REDIS_URL

File: app.json (hunk 1 above)
Comment (maxUo) [inline app.json:17]: STORAGE_POSTGRES => STORAGE_POSTGRES_URL

File: app.json (hunk 2 above)
Comment (maxUo) [inline app.json:21]: STORAGE_REDIS => STORAGE_REDIS_URL

File: app.json (hunk 1 above)
Comment (vikmeup) [reply app.json:17]: This is expected, `STORAGE_POSTGRES` will create `STORAGE_POSTGRES_` variables along with `STORAGE_POSTGRES_URL`

**Review by maxUo — APPROVED** [review]

**Pull request merged**